pages/folder/[[...id]]/index.tsx

```diff
@@ -71,14 +59,29 @@ export default function FolderPage() {
               결과입니다
             </p>
           )}
-          <FolderCategory folders={folders} selectedId={selectedId} />
-          <FolderControl folderName={selectedName as string} />
+          {folderListLoading ? (
+            <div>로딩중입니다...</div>
+          ) : folderListError ? (
+            <div>폴더 정보들을 가져오는데 실패했습니다.</div>
+          ) : (
```

Comment:
로딩, 에러 처리를 일단 위 같은 식으로 했는데 삼항연산자를 중첩해서 사용했는데 어떻게 개선하면 좋을까요?

Reply:
지금도 괜찮다고 생각하지만, 아래처럼 하는것도 괜찮을거같아요. status로, 같은 상태를 기준으로 하면서 &&로 짧은 표현으로요
```suggestion
         {status === 'loading' && <div>로딩중입니다...</div>}
         {status ===  'error' && 
            <div>폴더 정보들을 가져오는데 실패했습니다.</div>}
```